### DISCUSSION TITLE
Again timezones and surface format

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#2889] [Q&A] [answered]
I made a bug report #2796 of bad timezones. They have been fixed. And they don't exixt (anymore) in https://github.com/Stellarium/stellarium-data/blob/master/geonames/timeZones.txt.

I have originally installed Stellarium from (K)ubuntu repositories. Then I have used AppImage and now I have compiled Stellarium on my Kubuntu 22.10. I downloaded that timeZones.txt and copied to stellarium/data directory on my computer. And then to stellarium/build/unix/data/ directory. And configured compilation and compiled.

When I start this Stellarium (Stellarium 1.1 (v1.22.4-94821fc [master])), log says still bad timezones:

Loaded 252 countries
Loaded 193 regions
StelLocationMgr::loadCitiesBin(): TimeZone for  "Berehove"  not found:  "Europe/Uzhgorod"
StelLocationMgr::loadCitiesBin(): TimeZone for  "Khust"  not found:  "Europe/Uzhgorod"
StelLocationMgr::loadCitiesBin(): TimeZone for  "Mukachevo"  not found:  "Europe/Uzhgorod"
StelLocationMgr::loadCitiesBin(): TimeZone for  "Rakhiv"  not found:  "Europe/Uzhgorod"
StelLocationMgr::loadCitiesBin(): TimeZone for  "Svalyava"  not found:  "Europe/Uzhgorod"
StelLocationMgr::loadCitiesBin(): TimeZone for  "Thunder Bay"  not found:  "America/Thunder_Bay"
StelLocationMgr::loadCitiesBin(): TimeZone for  "Uzhgorod"  not found:  "Europe/Uzhgorod"
StelLocationMgr::loadCitiesBin(): TimeZone for  "Vynohradiv"  not found:  "Europe/Uzhgorod"
StelLocationMgr::loadCitiesBin(): Summary of unknown TimeZones:
"Europe/Uzhgorod"
"America/Thunder_Bay"
Please report these t

What is wrong?

Another thing: There is warning in log.txt file:

Default surface format:  QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize -1, stencilBufferSize -1, samples -1, 
swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 1, colorSpace QColorSpace(), profile  QSurfaceFormat::NoProfile)
OpenGL module type: 0
Warning: Setting a new default format with a different version or profile after the global shared context is created may cause issues with context sharing.
Default surface format:  QSurfaceFormat(version 3.3, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize 24, redBufferSize 8, greenBufferSize 8, blueBufferSize 8, alphaBufferSize -1, stencilBufferSize -1, samples -1, swa
pBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 1, colorSpace QColorSpace(), profile  QSurfaceFormat::CoreProfile)
OpenGL module type: 0

I have not found where is defined what format to use. Maybe this occurs when there are Kubuntu repository Stellarium and that creates .ini files. And then when using newest version makes that? The Stellarium works ok. So this is not a big problem. I want to learn more how Stellarium works.

**Comment by 10110111** [comment, marked as answer]
The fix done for #2796, 327a66c3e43545cfce77a238cf33fcf25ac71245, happens after your version v1.22.4-94821fc, and includes a change of a `.cpp` file, not `.txt`.

As for the GL format warning, it's issued by Qt, not Stellarium, and can be safely ignored if there are no issues with graphics.
